src/components/ChatLog.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './ChatEntry.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry.js';
+// import timestamp component
+
+const ChatLog = ({entries, toggleLike}) => {
+    const chatComponents = entries.map(chatInstance => {
```

Comment:
Nice use of map so we can map over each entry and render a unique component for each